src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -969,16 +968,17 @@ public void verifyContainsElement(By by) {
      * timing issues due to page load, inconsistencies in Selenium API, etc.
      */
     public void waitForTextsForAllStatusMessagesToUserEquals(final String... expectedTexts) {
+        Assumption.assertFalse(expectedTexts.length == 0);
```

Comment:
Prefer compile time failure to runtime failure?

Could something like this work:
`waitForTextsForAllStatusMessagesToUserEquals(String firstExpectedTexts, String ...remainingExpectedTexts)`